modules/luci-mod-status/root/usr/share/rpcd/acl.d/luci-mod-status.json

```diff
@@ -96,6 +96,7 @@
 		"description": "Grant access to main status display",
 		"read": {
 			"file": {
+				"/sys/fs/pstore/*": [ "read" ],
```

Comment:
Could you please move the acl into its on section `luci-mod-status-index-pstore`

Reply:
Yes, the description of the section i used ""Grant access to main status display"" was confusing